lib/galaxy/authnz/managers.py

```diff
@@ -304,10 +304,14 @@ def get_cloud_access_credentials(self, cloudauthz, sa_session, user_id, request=
             ca = CloudAuthz()
             log.info("Requesting credentials using CloudAuthz with config id `{}` on be half of user `{}`.".format(
                 cloudauthz.id, user_id))
-            return ca.authorize(cloudauthz.provider, config)
+            credentials = ca.authorize(cloudauthz.provider, config)
+            return credentials
         except CloudAuthzBaseException as e:
             log.info(e)
             raise exceptions.AuthenticationFailed(e)
+        except NotImplementedError as e:
+            log.info(e.message)
+            raise exceptions.RequestParameterInvalidException(e.message)
```

Comment:
Exceptions don't have the `message` attribute any more under Python 3
```suggestion
            raise exceptions.RequestParameterInvalidException(e)
```

Reply:
Good to know, thanks!